app/src/lib/stats/stats-store.ts

```diff
@@ -799,8 +786,7 @@ function createLocalStorageTimestamp(key: string) {
  * be converted into a number this method will return null.
  */
 function getLocalStorageTimestamp(key: string): number | null {
-  const value = parseInt(localStorage.getItem(key) || '', 10)
-  return isNaN(value) ? null : value
+  return getNumber(key) || null
```

Comment:
I don't believe it matters here since a time stamp shouldn't ever be zero but this will return `null` for both `0` and `undefined` due to the falsy-check whereas it should technically only return null when the value is undefined. If it helps I'd be fine with changing the return type to `number | undefined` and updating the call sites. Either that or we bring back the intermediate variable and check for undefined explicitly.

Reply:
Ugh, falsiness. Good spot